src/Sulu/Bundle/WebsiteBundle/Twig/Navigation/NavigationTwigExtension.php

```diff
@@ -56,6 +56,7 @@ public function getFunctions()
             new \Twig_SimpleFunction('sulu_navigation_flat', [$this, 'flatNavigationFunction']),
             new \Twig_SimpleFunction('sulu_navigation_tree', [$this, 'treeNavigationFunction']),
             new \Twig_SimpleFunction('sulu_breadcrumb', [$this, 'breadcrumbFunction']),
+            new \Twig_SimpleFunction('sulu_is_active_nav_element', [$this, 'isActiveNavElementFunction']),
```

Comment:
i would call it `sulu_navigation_is_active` to keep the prefix. additionally we don't use abbreviations.